src/ctelnet.cpp

```diff
@@ -1330,7 +1328,7 @@ void cTelnet::processTelnetCommand(const string& command)
                 packageName.remove(QLatin1Char('\\'));
                 packageName.remove(QLatin1Char('.'));
 
-                postMessage(tr("[ INFO ]  - Server offers downloadable GUI (url='%1') (package='%2')...").arg(url, packageName));
+                postMessage(tr("[ INFO ]  - Server offers downloadable GUI (url='%1') (package='%2').").arg(url, packageName));
```

Comment:
```suggestion
                postMessage(tr("[ INFO ]  - Server offers downloadable interface (url='%1') (package='%2').").arg(url, packageName));
```

Reply:
"GUI" does appear in many other places, would need adjustment everywhere to be consistent. :bulb: Not scope of this PR to improve translatability.

Reply:
Discuss it in issue #2260